tests/test-modules/107-scatterplot/app.R

```diff
@@ -0,0 +1,24 @@
+library(shiny)
```

Comment:
Make sure this app is tested. Or remove it.

Reply:
4299e0b0